synapse/handlers/auth.py

```diff
@@ -846,8 +846,9 @@ def hash(self, password):
             Deferred(str): Hashed password.
         """
         def _do_hash():
-            return bcrypt.hashpw(password.encode('utf8') + self.hs.config.password_pepper,
-                                 bcrypt.gensalt(self.bcrypt_rounds))
+            return bcrypt.hashpw(
+                password.encode('utf8') + self.hs.config.password_pepper,
+                bcrypt.gensalt(self.bcrypt_rounds))
```

Comment:
If we're reformatting, we prefer:

```
foo(
    1,
    2,
)
```

over 
```
foo(
    1,
    2)
```

(https://github.com/matrix-org/synapse/blob/develop/docs/code_style.rst has some words on this)